edgelet/dps/src/lib.rs

```diff
@@ -3,27 +3,14 @@
 #![deny(unused_extern_crates, warnings)]
```

Comment:
Replace `unused_extern_crates` with `rust_2018_idioms`. The "ideal" 2018 state is to not have any `extern crate` statements, so `unused_extern_crates` becomes useless.